tests/cython/run_test_cython.py

```diff
@@ -0,0 +1 @@
+from . import test_cython  # noqa: F401
```

Comment:
Possibly it makes sense to refactor test_cython to export a main function and we can then call it here

Reply:
Yeah, I thought so. But it was late and I was lazy :)

Will do!